### PR TITLE
Update mtf_scenariotest.md

### PR DESCRIPTION
**Pull request by mage2pratik**
## This PR is a:

- [ ] New topic
- [x] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will remove unnecessary space and update latest code for version 2.2 and 2.3.

## Additional information
N/A

### CODE REVIEW

**Comment by magento-cicd2** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md [hunk 1]
```diff
@@ -160,10 +158,9 @@ The example defines the following test steps and their sequence:
 
 ## Test step class  {#test-step}
 
-All test steps are located in `<module>/Test/TestStep` as PHP classes in the following format:
+All test steps are located in `<module>/Test/TestStep` as PHP classes with the following format:
```
Comment (hguthrie) [inline guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: Could be:
"The  `<module>/Test/TestStep` directory contains each test step as a PHP class with the following format:"

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md [hunk 2]
```diff
@@ -174,9 +171,9 @@ class YourTestStep implements TestStepInterface
 {
     public function __construct
     (
-        // data that are required for this step along with other dependencies.
+        // data that is required for this step, along with other dependencies.
```
Comment (hguthrie) [inline guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: data and other dependencies required for this step

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md [hunk 3]
```diff
@@ -174,9 +171,9 @@ class YourTestStep implements TestStepInterface
...
     ) {
-        // all required classes (fixtures, pages, etc.) have to be assigned here.
+        // all required classes (fixtures, pages, etc.) are assigned here.
```
Comment (hguthrie) [inline guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: assign required classes (fixtures, pages, etc.) here

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md [hunk 4]
```diff
@@ -298,32 +294,32 @@ class LoginCustomerOnFrontendStep implements TestStepInterface
  
 ### `constructor()` method  {#constructor-method}
 
-You can optionally use the `constructor()` method which injects data to be used by `run()`. The data may include classes like pages, variation data, data returned by previous executed steps, etc.
+You may optionally use the `constructor()` method which injects data to be used by `run()`. The data may include classes like pages, variation data, data returned by previous executed steps, etc.
```
Comment (hguthrie) [inline guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: splitting?

"Optionally, you may use..."

Comment (hguthrie) [reply guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: comma before "which"

Tells us what `run()` is like the constructor. Maybe "...the `run()` method."

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md (hunk 1 above)
Comment (dobooth) [reply guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: Thanks.

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: Right.

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: Thanks.

File: guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md (hunk 4 above)
Comment (hguthrie) [reply guides/v2.1/mtf/mtf_entities/mtf_scenariotest.md]: ^^

**Comment by dobooth** [comment]
running tests

**Pull request merged**